### DISCUSSION TITLE
Is there any way to change dx/dy of the CmdCreateAnnotCircle or CmdCreateAnnotSquare?

### DISCUSSION THREAD

**Discussion by scsmash3r** [sumatrapdfreader/sumatrapdf#3874] [Q&A]
So, the question is really about changing the dimensions of an objects, pasted as annotations: how to resize them?
I can see the params like `Rectangle x=100 y=100 dx=101 dy=51` at the Annotations panel, but they are static.

##### Additional question:
I've also managed to create the shortcut for Polygon below as is:
```
Shortcuts [
	[
		Cmd = CmdCreateAnnotPolygon
		Key = 1
	]
]
```
...but it seems it is not working at all. Is it not supported?

I'm on Windows 10/11, using `646d1fe` onboard. I was using SumatraPDF as my daily driver for years now, mainly for viewing PDFs. But now it came to editing, so I'm really digging what features are there and how to customize them, if possible.

**Comment by GitHubRulesOK** [comment]
resize is not yet implemented open issue https://github.com/sumatrapdfreader/sumatrapdf/issues/1665
polygon is not yet implemented open issue https://github.com/sumatrapdfreader/sumatrapdf/issues/2485

The annotation is Generated via MuPDF which has a MuPDF-GL version that allows to add and modify (its quirky but works)
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/51ddac2a-fc05-4845-a61f-40843d66e5a2)
